~

```diff
@@ -0,0 +1,6 @@
+Merge remote-tracking branch 'upstream/main' into issue-736/create-side-panel-testing
+# Please enter a commit message to explain why this merge is necessary,
+# especially if it merges an updated upstream into a topic branch.
+#
+# Lines starting with '#' will be ignored, and an empty message aborts
+# the commit.
```

Comment:
what file is this?

Reply:
Literally zero clue what this is -- will delet ⚡

Reply:
Looks like VIM output after merging branches